health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/MyMicroplans.js

```
@@ -0,0 +1,50 @@
+    <React.Fragment>
+
+      <Header styles={{ fontSize: "32px" }}>{t("MICROPLANS")}</Header>
+      <div className="inbox-search-wrapper">
+        {/* Pass defaultValues as props to InboxSearchComposer */}
+        <InboxSearchComposer
+          configs={config}
+          defaultValues={defaultValues}
+          showTab={true}
+          tabData={tabData}
+          onTabChange={onTabChange} //!activated ion tabChange
+        ></InboxSearchComposer>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)